adafruit_mcp9600.py

```diff
@@ -82,6 +193,77 @@ def __init__(self, i2c, address=_DEFAULT_ADDRESS, tctype="K", tcfilter=0):
         self.buf[1] = tcfilter | (ttype << 4)
         with self.i2c_device as tci2c:
             tci2c.write(self.buf, end=2)
+        if self._device_id != 0x40:
+            raise RuntimeError("Failed to find MCP9600 - check wiring!")
+
+    def alert_config(self, *, alert_number, alert_temp_source, alert_temp_limit, alert_hysteresis,
+                     alert_temp_direction, alert_mode, alert_state):
+        """Configure a specified alert pin. Alert is enabled by default when alert is configured.
+        To disable an alert pin, use ``alert_disable``.
+
+        :param int alert_number: The alert pin number. Must be 1-4.
+        :param alert_temp_source: The temperature source to monitor for the alert. Options are:
+                                  ``THERMOCOUPLE`` or ``AMBIENT``.
```

Comment:
The temperature source to monitor for the alert. Options are:
                                  ``THERMOCOUPLE`` (hot junction) or ``AMBIENT`` (cold junction). In Celsius.